### ISSUE TITLE
TL135 bulb

### ISSUE THREAD

**Issue by simpat1zq** [petretiandrea/home-assistant-tapo-p100#659]
**Is your feature request related to a problem? Please describe.**
I have a tapo TL135 bulb and it is not working and it's not listed in the devices list in the docs. 

**Describe the solution you'd like**
Any help adding it would be greatly appreciated. Please let me know if you need me to run the script below or not. I'm not sure if it's to discover a new bulb or not.  

**Describe alternatives you've considered**

**Device Info**
Please execute this python script. You can save it as `main.py` and run it by using `python main.py`.
Be sure to have the latest version of `plugp100` library, by installing it through `pip install plugp100`.

```python
import asyncio

from plugp100 import TapoApiClient, TapoApiClientConfig, LightEffect

async def main():
    # create generic tapo api
    config = TapoApiClientConfig("<ip>", "<email>", "<passwd>")
    sw = TapoApiClient.from_config(config)
    await sw.login()
    await sw.on()
    await sw.set_brightness(100)
    state = await sw.get_state()
    print(state.get_unmapped_state())

loop = asyncio.get_event_loop()
loop.run_until_complete(main())
loop.run_until_complete(asyncio.sleep(0.1))
loop.close()

if __name__ == "__main__":
    main()
```

**Comment by petretiandrea** [comment]
> TL135

You must run the script and report here the output

**Comment by simpat1zq** [comment]
I installed the plugp100 library and then ran that script, but I get an error:

...ImportError: cannot import name 'TapoApiClient' from 'plugp100'...

So I did some Googling, and found that in a previous ticket, they were asked to install "pip install plugp100==2.5.1"

So I tried that and reran the script. This time it gave me the same error for LightEffect. But the script that was posted with that version didn't have the LightEffect import in there. So I'm guessing there is a specific version of plugp100 I need for this version of your script. 

That said, I was in the Tapo app today, trying to add the bulb, and I noticed that the when you try to add the bulbs, this one is listed as "Tapo L535/TL135". and #657 is for the L535, and that looks like it's targeted for your next release. Does that mean that when that bulb is in there, that this one will work as well?

Just let me know if you need any other info from me.

**Comment by petretiandrea** [comment]
Try this
```python
import os

from plugp100.api.light_effect_preset import LightEffectPreset
from plugp100.api.tapo_client import TapoClient
from plugp100.common.credentials import AuthCredential
from plugp100.discovery.arp_lookup import ArpLookup
from plugp100.discovery.tapo_discovery import TapoDiscovery


async def main():
    # create generic tapo api

    credentials = AuthCredential("<tapo_email>", "<tapo_password>")
    client = TapoClient.create(credentials, "<tapo_device_ip>")

    print(await client.get_device_info())


if __name__ == "__main__":
    loop = asyncio.new_event_loop()
    loop.run_until_complete(main())
    loop.run_until_complete(asyncio.sleep(0.1))
    loop.close()
```

**Comment by esondag** [comment]
I am also interested in getting the TL135 bulbs working. I am not sure what to do to help, but I can figure it out with some guidance. Thank you for your consideration!

**Comment by petretiandrea** [comment]
Released on 2.14.0
